### DISCUSSION TITLE
Wrong colors in VR

### DISCUSSION THREAD

**Discussion by ResidentVR** [praydog/REFramework#522] [General Questions] [answered]
Hello. I have an issue with colors in VR Resident Evil 2 and Resident Evil 3. I use Oculus Quest 2 and Radeon RX 6700 XT. I've tried OpenVR and OpenXR, Link and Air Link connection, turning HDR on and off. Wrong colors are only visible in VR and look fine on my TV and virtual desktop. Is there anything I can do in the settings to fix this issue?
![re2_1](https://user-images.githubusercontent.com/113498547/190073800-6e8c8d4c-490c-42f0-81e5-8c917dc3484b.jpeg)
![re2_2](https://user-images.githubusercontent.com/113498547/190073813-f2a5f380-1399-4416-9552-b048d73578dd.jpeg)
![re3_1](https://user-images.githubusercontent.com/113498547/190073817-04f11d99-fc86-4a6a-9ada-3e54a439d79d.jpeg)
![re3_2](https://user-images.githubusercontent.com/113498547/190073820-10c9f92f-d37e-422b-8086-f70c69d7be72.jpeg)

**Comment by Jamavo83** [comment]
I just installed today, quest 2, same problem.  On an RX 6900 XT.  But I'm trying to play Resident Evil 7 instead.  Haven't tried it on any others yet.  Maybe I'll give 8 a try. I started laughing, thought it looked like what the predator would see.  Looks fine on the computer screen as well, just the HMD is on drugs. Any help would be greatly appreciated.

**Reply by Kirito0016** [reply]
Same here on Vega56 on OpenXR.

**Comment by nhac-lly** [comment]
Same, seems to be AMD stuff. RX 6800.

**Comment by ResidentVR** [comment, marked as answer]
I found a solution! I have a 4k HDR TV and a Full HD monitor. When I launch a game with the TV connected to my PC I get this bug with colors and recently I launched a game with the moninor connected and got no bug.

But what should you do if you don't have another monitor?

Just create a shortcut to a game on your desktop, select it, unplug the HDMI cable from your PC and press Enter on your keyboard to launch a game.

**Comment by Kirito0016** [comment]
I can confirm. It works in openvr_api! 
Only no sound on openxr. 
You can reconnect hdmi/dp cable after the game has started and viewable on headset.

Edit 5 hrs later:
Instead unplugging cable, I can just power off monitor, and start the game. (It works...)

**Comment by Kirito0016** [comment]
My Quest 1 has just updated, and I can play the game with QHD monitor connected and on. (playing on openvr api)
